### PR TITLE
Add link to logos

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by nilsdeppe — DISMISSED** [review]

File: docs/Main.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ style="max-width:100%;"></a>
 </p>
 \endhtmlonly
 
+![banner](docs/Images/banner.png)
```
Comment (knelli2) [inline docs/Main.md]: This actually prints the word "banner" under the image (and is probably the same in the README)

Comment (knelli2) [inline docs/Main.md]: Should the word "spectre" be a bit larger? It looks a tad small in the space that it's in.

**Review by knelli2 — CHANGES_REQUESTED** [review]

File: docs/Main.md (hunk 1 above)
Comment (nilsvu) [reply docs/Main.md]: 🤷‍♂️ I think it's fine, it's personal preference I suppose. Try this size and adjust later if we find it's too small?

FYI, I intend to add a gravitational wave visualization to this banner when I find the time, to have the "s" radiate a burst of GWs in 3D.

Comment (nilsvu) [reply docs/Main.md]: Fixed, thanks for catching

**Review by knelli2 — APPROVED** [review]
I think we'll want to change how this looks for dark-mode. The random white rectangle looks a bit odd. But we can do that later.

image

**Pull request merged**